src/parser.ml

```diff
@@ -7,12 +7,17 @@ let print_files sources target =
                  |-target: %s\n"
     (String.concat " / " sources) target
 
-
+(* Print wrong target error with cmdliner style *)
+let target_error target =
+  "DEST... arguments: no `"^target^"' file or directory\n\
+  \rUsage: distar [OPTION]... SOURCE... DEST\n\
```

Comment:
Put this part in a `usage` function as it does not only relate to the error message.